conda-recipe/meta.yaml

```diff
@@ -21,6 +21,13 @@ requirements:
    # this link needed quite some searching, please do not delete!
    # https://go.microsoft.com/fwlink/?LinkId=532606&clcid=0x409
 
+  build:
```

Comment:
This build section is repeated from above.  Should be deleted, right?

Reply:
Right. Don't know how that got cloned. Checking locally and then committing.